sites/platform/static/files/fetch/examples/relationships/rabbitmq

```diff
@@ -1,18 +1,22 @@
 {
     "username": "guest",
-    "scheme": "amqp",
-    "service": "rabbitmq39",
     "fragment": null,
     "ip": "169.254.220.199",
-    "hostname": "453lqa3dz2ekflsrm7whfdhbiq.rabbitmq39.service._.eu-3.platformsh.site",
-    "port": 5672,
     "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "rabbitmq.internal",
-    "rel": "rabbitmq",
     "path": null,
     "query": [],
     "password": "guest",
+    "port": 5672,
+    "host_mapped": false,
+    "service": "rabbitmq39",
+    "hostname": "453lqa3dz2ekflsrm7whfdhbiq.rabbitmq39.service._.eu-3.platformsh.site",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.rabbitmq39.service._.eu-1.platformsh.site",